sonaranalyzer-dotnet/rspec/cs/S4260_c#.html

```diff
@@ -0,0 +1,52 @@
+<p>When creating a custom Markup Extension that accepts parameters in WPF, the <code>ConstructorArgument</code> markup must be used to identify the
+discrete properties that match these parameters. However since this is done via a string, the compiler will not notice if there are typos.</p>
+<p>This rule raises an issue when the string argument to <code>ConstructorArgument</code> don't match any of the constructor's parameters.</p>
```

Comment:
Small tweak to wording: "This rule raises an issue when **a** string argument to <code>ConstructorArgument</code> **doesn't** match any of the constructor's parameters."

Reply:
Ahaha cool I can say to Jesse he made English spelling errors!

Reply:
To be fair, I'm not 100% on a/the here. If only we had native speaker on the team... @duncanp-sonar